pkg/asset/installconfig/powervs/session.go

```diff
@@ -182,6 +184,62 @@ func (c *BxClient) ValidateAccountPermissions() error {
 	return nil
 }
 
+//ValidateDhcpService checks for existing Dhcp service for the provided PowerVS cloud instance
+func (c *BxClient) ValidateDhcpService(ctx context.Context, svcInsID string) error {
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	defer cancel()
+
+	//create Power VS DHCP Client
+	dhcpClient := instance.NewIBMPIDhcpClient(ctx, c.PISession, svcInsID)
+	//Get all DHCP Services
+	dhcpServices, err := dhcpClient.GetAll()
+	if err != nil {
+		return errors.Wrap(err, "failed to get DHCP service details")
+	}
+	if len(dhcpServices) > 0 {
+		return fmt.Errorf("DHCP service already exists for provided cloud instance")
+	}
+	return nil
+}
+
+//ValidateCloudConnectionInPowerVSRegion counts cloud connection in PowerVS Region
+func (c *BxClient) ValidateCloudConnectionInPowerVSRegion(ctx context.Context, svcInsID string) error {
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	defer cancel()
+	var cloudConnectionsIDs []string
+	cloudConnectionClient := instance.NewIBMPICloudConnectionClient(ctx, c.PISession, svcInsID)
+
+	//check number of cloudconnections
+	getAllResp, err := cloudConnectionClient.GetAll()
+	if err != nil {
+		return errors.Wrap(err, "failed to get existing Cloud connection details")
+	}
+
+	if len(getAllResp.CloudConnections) >= 2 {
+		return fmt.Errorf("cannot create new cloud connection in powervs zone,only 2 cloud connections allowed")
```

Comment:
i'm sorry to keep picking on grammar, but this is a run-on sentence and needs proper nouns to be capitalized. "cannot create new Cloud connection in Power VS. Only two Cloud connections are allowed per zone."